src/index.ts

```diff
@@ -63,14 +95,19 @@ registerSessionsRoute(router)
 registerWelcomeRoute(router)
 registerErrorRoute(router)
 
+// all routes registered after this point are protected by CSRF
```

Comment:
Don't middlewares apply to all routes that match their path? https://expressjs.com/en/guide/using-middleware.html#middleware.application

E.g. you could scope the middlewares to only the consent route by adding a path as the first parameter to the `app.use` call.